zebra-state/src/service.rs

```diff
@@ -50,15 +57,20 @@ enum ValidateContextError {
 }
 
 impl StateService {
+    const PRUNE_INTERVAL: Duration = Duration::from_secs(30);
```

Comment:
What's the reasoning behind this interval?

The target block interval is 75 seconds, so when we've done the initial sync, we'll be doing 2 prunes for every new block. That seems a bit redundant?

Reply:
The prune interval is just meant to defend against extremely frequent scans of the entire list of awaited utxos, when we're done with the initial block sync I think its fine if it prunes every time, because those prunes will be very cheap relatively speaking, as there will likely be no waiters and they will still be called very infrequently.